src/prefect/environments/storage/docker.py

```diff
@@ -23,6 +23,14 @@
     import docker
 
 
+def multiline_indent(string: str, spaces: int) -> str:
```

Comment:
This is reproduced from https://github.com/PrefectHQ/prefect/pull/3632/files#diff-4c6587fe7d1728fde257726e6d238f5f098c62f56cf2226c013642ae2063a5f7R22 and would be useful as a general util for formatting

Reply:
Until it's needed elsewhere, I'd prefer to keep this here. There's a mental indirection cost to having to look up a utility in a different file, and this one's simple enough that pulling it out doesn't feel worth it.